doc/user_guide/configuration.rst

```diff
@@ -107,6 +107,19 @@ and an optional API specific section, for example for the
    COMPUTE_ANNEXED_DOFS = false
    supported_fortran_datatypes = real, integer, logical
    default_kind = real: r_def, integer: i_def, logical: l_def
+   precision_map = i_def: 4,
```

Comment:
You also need to add a description of this new entry to the table under the `Dynamo 0.3` section below.

Reply:
Will I have to add something to the `doc/user_guide/dynamo0p3.rst` 'Configuration' section as well? The table you mentioned contains a small amount of information for each entry, which then link to the a more detailed description in the 'Configuration' section of the `doc/user_guide/dynamo0p3.rst` file.